test/integration/PublishEditConsume.test.ts

```diff
@@ -402,31 +405,32 @@ describe('Publish consume test', async () => {
   // )
   // assert(ipfsOrderTx, 'Ordering ipfs dataset failed.')
```

Comment:
I remember we had a discussion regarding uncommenting the IPFS tests which were failing before, did it occur again the behaviour? What error did we encounter on this test?
I think we can uncomment since the test above publishes the IPFS asset